invokeai/backend/model_manager/load/model_loaders/flux.py

```diff
@@ -193,7 +198,30 @@ def _load_from_singlefile(
             sd = load_file(model_path)
             if "model.diffusion_model.double_blocks.0.img_attn.norm.key_norm.scale" in sd:
                 sd = convert_bundle_to_flux_transformer_checkpoint(sd)
+            futures: list[torch.jit.Future[tuple[str, torch.Tensor]]] = []
+            # For the first iteration we are just requesting the current size of the state dict
+            # This is due to an expected doubling of the tensor sizes in memory after converting float8 -> float16
+            # This should be refined in the future if not removed entirely when we support more data types
+            sd_size = asizeof.asizeof(sd)
```

Comment:
We typically just do `x.nelement() * x.element_size()` to calculate the size of tensors in a model or state_dict. Is there a reason for using `asizeof(...)` here?

My main concerns with using asizeof are:
- Currently this is the only reference to pympler in our codebase. So, it looks like we might be able to drop it from our direct dependency list if we don't use it here.
- I suspect that it's slower (not tested though).

Reply:
updated to use this instead of asizeof